### PR TITLE
commit blog: 用 Wechaty 实现微信任务小助手

### PR DESCRIPTION
**Pull request by darkli**
Fixes # .

Make sure these boxes are checked before creating your pull request -- thank you!

- [ ] Add Blog Header, including title, author, date.
e.g.

``` md
---
 title: "'Score Your Face Photo' a ML&Wechaty practice"
 date: 2017-09-18 09:00 +0800
 author: huyingxi
 ---
 > Author: [@huyingxi](https://github.com/huyingxi/wechaty_selfie) enjoying ML&Wechaty at BUPT
```

- [ ] Keep all filenames & url as lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor` instead of `2017-10-06-WeChat PC Impactor`,
- [ ] Embed the photo & video before publishing, save all external file to the blog `/download/2018` directory.
- [ ] Add `<!--more-->` section to just show the abstract in blog homepage.
- [ ] Add teaser image for the blog
- [ ] Add your self as an author to the author data file at <https://github.com/wechaty/wechaty.github.io/blob/master/_data/authors.yml> with your GitHub username. And please also remember to reference yourself with the Github username in your blog post, so that we can have nice author information besides your blog after we published it!

[See more](https://github.com/wechaty/wechaty.github.io#contribute-guideline)

### CODE REVIEW

File: _posts/2020-07-17-wechaty-assistant.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+  - wechaty
+  - wechaty-puppet-padplus
+  - wechat-robot
+  - teamin
+  - chatbot
+header:
+   teaser: /assets/2020/wechaty-teamin-chatbot/home-01.jpg
+---
+
+<!-- markdownlint-disable -->
```
Comment (lijiarui) [inline _posts/2020-07-17-wechaty-assistant.md]: Please remove this: `<!-- markdownlint-disable -->`

**Review by lijiarui — CHANGES_REQUESTED** [review]
Good job, please change as I request and make ci green. Then I can merge this PR.

**Comment by lijiarui** [comment]
Also, please sign the CLA first.

**Comment by darkli** [comment]
提交blog的时候，用的身份不是darkli，无法做CLA，撤销这次PR，重新来一次

**Pull request closed without merging**